src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -62,10 +63,14 @@ public void run(String [] args) {
             System.exit(1);
         }
 
+    }*/
+    
+    public static void setContext(Controller controller, View view) {
```

Comment:
We already talked about this offline, but remember that using the `static` keyword to fix random bugs is almost always a symptom of something worse going on under the hood. So when I see code that uses the `static` keyword in a weird place like this, I'm immediately suspicious.